new/LIT7036Gabreelbeta.xml

```diff
@@ -0,0 +1,79 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>               
+                <p>This <term key="Qene">qǝne</term> poem attested in <ref type="mss" corresp="Tanasee6"/>, 
+                    readable only in part, is of the type <foreign xml:lang="gez">wāzemā</foreign>. 
+                    It is attributed to <persName ref="PRS14503GabraMarQene">mamhǝr Gabra Māryām of Kǝbrān</persName>
+                    (as <ref type="work" corresp="LIT7034Lateamar"/>).</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Wazema"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-07-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7036Gabreelbeta" passive="Tanasee6"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7036Gabreelbeta" passive="PRS14503GabraMarQene"/>
+                </listRelation>
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The text is taken from <ref type="mss" corresp="Tanasee6"/> 
+                            (division into verses is not indicated).</note>
+                        <ab>                                                  
+                            <l n="1">ገብርኤል፡ ቤተ፡ ዚአከ፡ መርዓተ፡ ዘዮሐንስ፡ አብ፡ ዘለነ፡ ገጻ፡ ፀሐይ፡</l>
```

Comment:
I read ስነ፡ ገጻ

Reply:
Yes, true.

Reply:
Actually, ዘለነ makes sense as well, can you clearly see ለ or you interprete?

Reply:
"which her face is the sun for us", though it is difficult to link it to what is before and after.

Reply:
Ok, it is rather ሰ than ለ, I will correct.